snuba/cli/multistorage_consumer.py

```diff
@@ -234,11 +248,15 @@ def multistorage_consumer(
             input_block_size=input_block_size,
             output_block_size=output_block_size,
             metrics=metrics,
+            producer=dead_letter_producer,
+            topic=dead_letter_queue,
         ),
     )
 
     def handler(signum: int, frame: Any) -> None:
         processor.signal_shutdown()
+        if dead_letter_producer:
+            dead_letter_producer.close()
```

Comment:
Is this not already in https://github.com/getsentry/snuba/pull/2437 (also in the version that works).
Is this PR based on the right branch ?